styles/global.css

```diff
@@ -141,7 +141,24 @@ em-emoji-picker {
   pre code {
     --at-apply: bg-transparent px0 py0 rounded-none leading-1.6em;
   }
-
+  ol {
+    --at-apply: list-decimal my-3 pl-6 ml-2;
+  }
+  ul {
+    --at-apply: list-disc my-3 pl-6 ml-2;
+  }
+  li {
+    --at-apply: mt-1 mb-1;
+    &:empty {
+      --at-apply: hidden;
+    }
+  }
+  'ol > li' {
```

Comment:
@danielroe is this intentional? I got this
<img width="774" alt="image" src="https://user-images.githubusercontent.com/11247099/212670270-ad66b236-4153-4d4a-aa88-27b437efdea6.png">